src/tsconfig.json

```diff
@@ -0,0 +1,65 @@
+{
+    "compilerOptions": {
+      "noEmit": true,
+      /* Basic Options */
+      // "incremental": true,                   /* Enable incremental compilation */
+      "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```

Comment:
We want to publish modern code so the user can decide if they need to transpile the code. Next.js and CRA now transpile node modules, and Next.js even provides modern bundles for newer browsers.